### PR TITLE
Updates the local kubecontext with that of sandbox and switches to it

### PR DESCRIPTION
**Pull request by pmahindrakar-oss**
Signed-off-by: Prafulla Mahindrakar <prafulla.mahindrakar@gmail.com>

# TL;DR
Current way of using the sandbox kube context is by passing the kubeconfig as parameter

```
kubectl get pods -n flyte  --kubeconfig   /Users/praful/.flyte/k3s/k3s.yaml
```

This PR add the kube context created by k3d in sandbox environment in the local kube context and switches over to it so that users can monitor and check the status of the sandbox directly

```
kubectl get pods -n flyte 
```

It creates a new kube-context with name flyte-sandbox 
Any existing context with the same name would be overwritten

Testing Done:

Started the sandbox and monitored the flyte pods
```
(flyte-started) (base) ➜  flytectl git:(pmahindrakar/1604) ✗ kubectl config current-context
flyte-sandbox
(flyte-started) (base) ➜  flytectl git:(pmahindrakar/1604) ✗ git status                                                                                             
(flyte-started) (base) ➜  flytectl git:(pmahindrakar/1604) ✗ kubectl get pods -n flyte
NAME                                          READY   STATUS      RESTARTS   AGE
flyte-contour-contour-7cfc9f6fb5-8dv4s        1/1     Running     0          7m12s
flyte-kubernetes-dashboard-7fd989b99d-xdj67   1/1     Running     0          7m12s
flyteconsole-7c7698b9fc-vpqdl                 1/1     Running     0          7m12s
postgres-6485776865-mfmqc                     1/1     Running     0          7m12s
flytepropeller-6fdf748d5-dgxkt                1/1     Running     0          7m12s
minio-65fbb69485-4lrvq                        1/1     Running     0          7m12s
flyte-pod-webhook-67446897d6-td8kn            1/1     Running     0          7m12s
datacatalog-589d78fc57-bbdrr                  1/1     Running     0          7m12s
flyte-contour-envoy-trblm                     2/2     Running     0          7m12s
flyteadmin-bc9bd4cb9-hn6jf                    2/2     Running     0          7m12s
flytescheduler-795fdf957f-9btzd               1/1     Running     0          7m12s
syncresources-27349085-trflc                  0/1     Completed   0          2m10s
syncresources-27349086-vfqs6                  0/1     Completed   0          70s
syncresources-27349087-kcclj                  0/1     Completed   0          10s
```

### Sandbox teardown removes the context and unsets the current context

```
(base) ➜  flytectl git:(pmahindrakar/1604) ✗ flytectl sandbox teardown
(base) ➜  flytectl git:(pmahindrakar/1604) ✗ kubectl config current-context
flyte-sandbox
(base) ➜  flytectl git:(pmahindrakar/1604) ✗ flytectl sandbox teardown
context removed for "flyte-sandbox".
🧹 🧹 Sandbox cluster is removed successfully. 
(base) ➜  flytectl git:(pmahindrakar/1604) ✗ kubectl config current-context     
error: current-context is not set
```

## Type
- [ ] Bug Fix
- [X] Feature
- [ ] Plugin

## Are all requirements met?

- [X] Code completed
- [X] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1604

## Follow-up issue
_NA_

### CODE REVIEW

**Review by EngHabu — COMMENTED** [review]
Shouldn't `flytectl sandbox teardown` remove that context from local config?

**Comment by pmahindrakar-oss** [comment]
> Shouldn't `flytectl sandbox teardown` remove that context from local config?

Done @EngHabu

**Review by EngHabu — APPROVED** [review]
Awesome!
